stream-projects/do-something/shared/hooks/use-alert-connection.ts

```diff
@@ -0,0 +1,36 @@
+import { useComputed, usePollingQuerySignal } from "../../deps.ts";
+import { ALERT_CONNECTION_SUBSCRIPTION } from "../gql/alert-subscription.ts";
+
+const QUERY_POLL_INTERVAL = 1000;
+
+interface DoSomethingAlert {
+  id: string;
+  time: string;
+  data: {
+    user: {
+      name: string;
+      avatarUrl: string;
+    };
+    collectible: {
+      name: string;
+      iconUrl: string;
+    };
+  };
+}
+
+interface AlertResponse {
+  alertConnection: {
+    nodes: DoSomethingAlert[];
+  };
+}
+
+export function useAlertConnection() {
```

Comment:
I'd rename to `useAlerts$`

Reply:
Update: https://github.com/trufflehq/truffle-packages/pull/51/commits/0f3ae772e996fe63b9efdb07086d8514ce2b23e9